### ISSUE TITLE
CALL parsing erroring for MariaDB

### ISSUE THREAD

**Issue by Quinncuatro** [taozhi8833998/node-sql-parser#107]
```
# a.sql
USE databaseName;

CALL utility.create_backup_script(schemaName, 'table_name', 'ticket_name');

DELETE FROM table_name WHERE id = 0;
```

outputs

```
FILE ERRORED: a.sql
TYPE:         SyntaxError
MESSAGE:      Expected "#", "--", "/*", ":=", "=", or [ \t\n\r] but "u" found.
LOCATION:     {"start":{"offset":6,"line":2,"column":6},"end":{"offset":7,"line":2,"column":7}}
EXPECTED:     [{"type":"class","parts":[" ","\t","\n","\r"],"inverted":false,"ignoreCase":false},{"type":"literal","text":"/*","ignoreCase":false},{"type":"literal","text":"--","ignoreCase":false},{"type":"literal","text":"#","ignoreCase":false},{"type":"literal","text":":=","ignoreCase":false},{"type":"literal","text":"=","ignoreCase":false},{"type":"class","parts":[" ","\t","\n","\r"],"inverted":false,"ignoreCase":false},{"type":"literal","text":"/*","ignoreCase":false},{"type":"literal","text":"--","ignoreCase":false},{"type":"literal","text":"#","ignoreCase":false},{"type":"literal","text":":=","ignoreCase":false},{"type":"literal","text":"=","ignoreCase":false}]
FOUND:        u
AST:          CALL utility.create_backup_script(schemaName, 'table_name', 'ticket_name');
DELETE FROM table_name WHERE id = 0;
```

**Comment by Quinncuatro** [comment]
Seems more to be an issue if there's a blank line before the first parsable line.

I have a script to pull out lines I don't care about (`USE databaseName`) in this case. So there will be an empty line above the `CALL` line. If I manually remove that blank line between `USE` and `CALL` before running it through my sanitizer to pull out lines I don't care about, I get the above error.

**Comment by taozhi8833998** [comment]
fixed in `1.6.6` version
